### DISCUSSION TITLE
Adding FEC problem

### DISCUSSION THREAD

**Discussion by bulerd** [Mr-MIBonk/M.I.B._More-Incredible-Bash#171] [Q&A]
Hi everyone. So rookie about MIB solutions i am and  i 've read all topics in discussion to find a solution to my problem but i couldn't. I am so sure, it is very simple for you and you will think how silly i am. Anyway..

I 've been trying to update maps. But because of 023000ee issue, i couldn't. I have MHI2_ER_AUG22_P3241_MU0678 and GEM 4.5 installed with M.I.B_patches_2023_04_08. Problem is, i couldn't add FEC because i don't know how to do. How will i add FEC to FECContainer.fec via GEM or in other words how will i be able to add  023000ee while mine is 023d001e. By the way i am trying to update Turkey Map...

Thanks for help.....

**Comment by bulerd** [comment]
Installing different toolbox problem it was. I installed yours and problem was solved. Thank you very much. World is much more beautiful because of you...

**Comment by bulerd** [comment]
Results after installation;

023000ee----o.k
P101_N60S5MIBH3_ROW from MIB Solutions---couldn't be updated. :(
Car Settings in MIH2----Disabled :(
Information ( Media-Telephone-Navigation) in Virtual Cockpit---Disabled  :)

[MU0678-JTB-07423.04.161183030B.log](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/files/11185573/MU0678-JTB-07423.04.161183030B.log)


Please help me ....

**Comment by bulerd** [comment]
Problem is solved.

**Reply by ERRAYND** [reply]
How, I have same problem
